.env

```
@@ -4,7 +4,7 @@
 TILESET_FILE=openmaptiles.yaml
 
 # Use 3-part patch version to ignore patch updates, e.g. 5.0.0
-TOOLS_VERSION=5.3
+TOOLS_VERSION=5.3.2
```

I believe that tools 5.3 including minor version 5.3.2

```suggestion
TOOLS_VERSION=5.3
```